addons/mail/static/src/discuss/call/common/rtc_service.js

```diff
@@ -639,6 +641,11 @@ export class Rtc extends Record {
         }
     }
 
+    async toggleCameraFacingMode() {
+        this.state.frontFacingMode = this.state.frontFacingMode ? false : true;
```

Comment:
```suggestion
        this.state.frontFacingMode = !this.state.frontFacingMode
```
Isn't this equivalent ?